src/css/skins/seven.less

```diff
@@ -54,4 +64,38 @@
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
   }
+
+  /* Next up display */
+  .jw-nextup-container {
+    bottom: @nextup-bottom-position;
+  }
+  .jw-nextup-header {
+    background-color: @nextup-header-background;
+    color: @accent-color;
+  }
+  .jw-nextup {
+    border-radius: .3em;
+  }
+  .jw-nextup-body {
+    background: @nextup-body-background;
+  }
+  .jw-nextup-thumbnail-visible + .jw-nextup-title::after {
+    background: linear-gradient(
+      -180deg,
+      fade(@nextup-title-overflow-background, 0%) 0%,
+      @nextup-title-overflow-background 100%
+    );
+  }
+  .jw-nextup-title {
```

Comment:
Something I need to do is get more familiar with how this code base works, but this and @robwalch's comment helped and the changes I made should be closer to what you had in mind.